az-digital/az_quickstart
# Closes #2167 AZ Person 'Link to external bio' is not currently implemented

## Problem/Motivation
While getting the president's site ready for launch we noticed that people in the grid view were not linking to their external bio pages.



## Proposed resolution
Use `[node:az-canonical-url]` as a custom URL for the link group on card or other listing view modes.




## Description
We found that the az_card display mode does not use the "link to external bio" field.

On the president's site, I changed the az_card display to use `[node:az-canonical-url]`on `/admin/structure/types/manage/az_person/display/az_card` under `group_link`
![image](https://user-images.githubusercontent.com/1023167/213790277-c9ac12b0-b9cb-400b-856a-6737737f5c1c.png)

The above action fixed the issue.


## Related issues
Closes #2167 
Blocked by https://github.com/az-digital/az_quickstart/pull/2077

## How to test


1. Go to `/node/add/az_person`
2. Fill out fields as you would normally.
3. Click on 'Summary Fields'
![image](https://user-images.githubusercontent.com/1023167/213789511-e7c4eddd-8b21-4994-af7a-992841743534.png)

4. Add a link to a separate (external) profile.
5. Save
6. Go to `/people`
7. See link to external bio being used.


## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
